Discovery/Discovery/Presentation/WebDiscovery/DiscoverWebviewModel.swift

```diff
@@ -0,0 +1,235 @@
+//
+//  DiscoverWebviewModel.swift
+//  Discovery
+//
+//  Created by SaeedBashir on 12/16/23.
+//
+
+import Foundation
+import Core
+import WebKit
+import SwiftUI
+import Swinject
+
+public class DiscoveryWebviewModel: ObservableObject {
```

Comment:
What about DiscoveryWebViewModel or DiscoveryWebviewViewModel?
To emphasize more explicitly that this is a ViewModel for the DiscoveryWeb feature.

Reply:
Done